floris/tools/floris_interface.py

```diff
@@ -642,6 +662,23 @@ def get_turbine_ais(self) -> NDArrayFloat:
     def turbine_average_velocities(self) -> NDArrayFloat:
         return average_velocity(velocities=self.floris.flow_field.u)
 
+    @property
+    def turbine_effective_velocities(self) -> NDArrayFloat:
```

Comment:
@rafmudaf Added `turbine_effective_velocities` at the `floris_interface` level. Should this be an `@property`? Modelled after `turbine_average_velocities`.